src/python/library/requirements/requirements_http.txt

```diff
@@ -25,6 +25,6 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 geventhttpclient>=1.4.4,<=2.0.2
-aiohttp>=3.8.1
+aiohttp>=3.8.1,<=4.0.0
```

Comment:
Is there a reason for including version `4.0.0`? I though it is the version that errored in Python 3.10?

Reply:
Sorry, wrongly copied it...